internal/live/live_test.go

```
@@ -0,0 +1,749 @@
+				case "hb":
+					hb++
+					if hb == 2 && !ackSeen {
+						// Wake the client with a real response, then demand an ack.
+						if err := writeFrameBytes(bw, 0x2, responsePB([][]byte{
+							messagePB(methodChat, chatPB(testUser(), "hb-woken")),
+						}, "c2", 0, "acktok-7")); err != nil {
+							return err
+						}
+					}
+				case "ack":
+					ackSeen = true
```

golangci-lint (2.12.2)

[error] ineffectual assignment to ackSeen

(ineffassign)